docker/monitor/bootstrap-table.min.css

```diff
@@ -7,4 +7,387 @@
   * @license MIT
```

Comment:
these are  minimized CSS, so on purpose all spaces are removed. Anything with .min. in the filename should be left as is.

Reply:
This can be solved easily.